sklearn/preprocessing/discretization.py

```diff
@@ -0,0 +1,228 @@
+#title          :discretization.py
+#description    :Implementation of a few discretization algorithms
+#author         :Henry Lin <hlin117@gmail.com>
+#==============================================================================
+
+from __future__ import division
+from math import log
+import numpy as np
+from scipy.stats import entropy
+from sklearn.base import BaseEstimator
+
+class MDLP(BaseEstimator):
+    """Implements the MDLP discretization criterion from Usama Fayyad's
+    paper "Multi-Interval Discretization of Continuous-Valued Attributes
+    for Classification Learning"
+    """
+
+    def __init__(self, **params):
```

Comment:
Thanks for the tip. I think I got confused by the documentation of the `OneHotEncoder`<a href="http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.OneHotEncoder.html">here</a>, whose `set_params()` function takes `**`. But now that I know that the `OneHotEncoder` just inherits that function from its base class, which has the `**` parameters, this makes sense to me.